assets/performanceprofile/configs/openshift-workload-pinning

```diff
@@ -0,0 +1,7 @@
+{{- if . }}
```

Comment:
And this is kubernetes config file?

Reply:
Right, this is the file we use as a flag to singal to kubelet